pandas/tseries/offsets.py

```diff
@@ -112,6 +112,24 @@ def wrapper(self, other):
     return wrapper
 
 
+def _make_cmp_func(op):
+    assert op not in [operator.eq, operator.ne]
+    # __eq__ and __ne__ have slightly different behavior, returning
+    # False and True, respectively, instead of raising
+
+    def cmp_func(self, other):
+        if type(self) == Week and self.weekday is None:
+            # Week without weekday behaves like a Tick
```

Comment:
blank line

Reply:
Sure, but for clarification, I thought that wasn't necessary when the pre-comment line has different indentation.  Is there a reference for this I can bookmark?